### DISCUSSION TITLE
searx

### DISCUSSION THREAD

**Discussion by ghost** [linuxserver/Heimdall#881] [Search Providers]
please add searx as a search engine to choose from. info about searx: 
[htps://searx.github.io/searx/](url)

list of public instances: 
[https://searx.space/](url)

personally i like [https://northboot.xyz/](url) for a german location (or close). for a faster search.

another good one would be [https://search.ononoki.org/](url) for the US. 

there are many different ones so maybe you can add searx and then let people choose from like a list of their favorite public searx instances.

**Comment by Dictynna-PHA** [comment]
searx:
   id: searx
   url: https://search.ononoki.org/search
   name: SearX
   method: get
   target: _blank
   query: q

**Reply by Scarletvye** [reply]
SearXNG alternative

searxng:
   id: searxng
   url: https://searx.work/search
   name: SearXNG
   method: post
   target: _blank
   query: q
